sections/dashboard/TrendingSynths/utils.ts

```diff
@@ -21,12 +21,12 @@ export const toCurrencyKeyMap = (
 	}, {});
 
 export const numericSort = (
-	comparatorMap: Record<CurrencyKey, NumericValue>,
+	comparatorMap: Partial<Record<CurrencyKey, WeiSource>>,
```

Comment:
same q: why partial? I see it semi-often

Reply:
normally Record expects to be a complete accounting of the type if it has limited values. As CurrencyKey has been converted into a set of valid strings, and this function may only take in a subset of those values, Partial needs to be added here.